CatLover.Api/services/common.service.go

```diff
@@ -0,0 +1,23 @@
+package services
+
+import (
+	"log"
+
+	"catlover.api/models"
+)
+
+func ReturnResponse(status string, code int32, message string, data interface{}) models.Response {
```

Comment:
I think code would be better if it was uuid - WDYT?

Reply:
I'm sorry but i don't understand the question. Do you mean GUID for primary keys in my tables?